picard/ui/mainwindow.py

```diff
@@ -944,6 +948,10 @@ def update_selection(self, objects=None):
                         }
                     self.set_statusbar_message(msg, mparms, echo=None,
                                                history=None)
+            elif isinstance(obj, Album):
+                obj.update_metadata_images()
```

Comment:
Hmm, the parent of a file is the track, not the album, right? I think I prefer your previous solution (emitting a signal from track/files and have the album's update_metadata_images called). Would that be ok?